microstream-cache/src/main/java/io/micronaut/microstream/cache/CacheConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.microstream.cache;
+
+import io.micronaut.core.util.Toggleable;
+
+/**
+ * Global Microstream Cache configuration.
+ * @author Sergio del Amo
+ * @since 1.0.0
+ */
+public interface CacheConfiguration extends Toggleable {
```

Comment:
I'm still confused. I still don't see the point of this interface, since it is empty and only has one implementation. Do you guys foresee an additional implementation in the future? 

Moreover, it can easily be confused with `one.microstream.cache.types.CacheConfiguration`.

Reply:
having this interface and class generates a configuration reference entry in the docuemntation. It exists to have better documentation. 

The point is to allow users to disable to whole cache module.

Reply:
We can delete `io/micronaut/microstream/cache/CacheConfiguration.java` and `io/micronaut/microstream/cache/CacheConfigurationProperties.java` but we will have to harcode the how to disable the module configuration in the docs.

Reply:
<img width="1213" alt="Screenshot 2022-05-09 at 22 06 48" src="https://user-images.githubusercontent.com/864788/167489006-eea90dee-f616-4b3b-8476-40f432349027.png">